### PR TITLE
Corporate form and page

### PR DESCRIPTION
**Pull request by diddydeeable**
## Description
Corporate page with the form to submit your details

it's not the most beautiful it can be - but I didn't want to spent too much time on styling as we have many more components to build 

next step will be to wire it up so that we can send data to db from this form

Screenshot 2024-07-31 at 13 03 26

## How Can This Be Tested?
- npm run dev
- go to the url : `http://localhost:5173/corporate`

## Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [x ] My changes generate no new warnings, including `` npm run build ``
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)

### CODE REVIEW

**Review by oskarprzybylski23 — APPROVED** [review]
LGTM

**Review by oskarprzybylski23 — CHANGES_REQUESTED** [review]
I've just noticed: Form has "enter password" field. This shouldn't be on contact form. Can you please amend and will then merge.

File: src/routes/AppRoutes.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import SignUp from '../pages/SignUp/SignUp';
 import About from '../pages/About/About';
 import ContactUs from '@/pages/ContactUs/ContactUs';
 import FAQs from '@/pages/FAQs/FAQs';
+import Corporate from '@/pages/Corporate/Corporate';
```
Comment (halimahexe) [inline src/routes/AppRoutes.tsx:8]: This has been added twice - please could you delete one instance?

File: src/routes/AppRoutes.tsx [hunk 2]
```diff
@@ -18,6 +19,7 @@ function AppRoutes() {
 			<Route path='/about' element={<About />} />
 			<Route path='/contact' element={<ContactUs />} />
 			<Route path='/faq' element={<FAQs />} />
+			<Route path='/corporate' element={<Corporate/>}/>
```
Comment (halimahexe) [inline src/routes/AppRoutes.tsx:22]: this already exists on line 24 - please could you delete one instance?

File: src/pages/Corporate/Corporate.tsx [hunk 3]
```diff
@@ -1,5 +1,68 @@
...
+				<div className='pt-8'>
+					<h3 className='text-3xl font-bold font-playfair pb-8 text-lilac'>
+						Our feedback
+					</h3>
+					<h2 className='font-source-sans text-xl pb-8'>
+						Here are some quotes from our most recent events
+					</h2>
+					<div className='grid grid-cols-2 pb-8'>
+						<div>REVIEW CARD HERE</div>
+						<div>REVIEW CARD HERE</div>
```
Comment (halimahexe) [inline src/pages/Corporate/Corporate.tsx:60]: Once we've merged the testimonials cards, we can add them here!

**Review by halimahexe — CHANGES_REQUESTED** [review]
Looks good - requested a few small changes as the repetition on AppRoutes threw an error!

File: src/routes/AppRoutes.tsx (hunk 1 above)
Comment (diddydeeable) [reply src/routes/AppRoutes.tsx:8]: I think I deleted this!

**Review by halimahexe — APPROVED** [review]
Thanks for making those changes. LGTM!

**Review by oskarprzybylski23 — APPROVED** [review]
LGTM

**Pull request merged**